cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -254,6 +239,19 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate rules
         printSummary(evaluatorRun.violations)
 
         outputDir?.let { absoluteOutputDir ->
+            val outputFiles = mutableSetOf<File>()
+
+            outputFormats.mapTo(outputFiles) { format ->
+                absoluteOutputDir.resolve("evaluation-result.${format.fileExtension}")
+            }
+
+            if (!globalOptionsForSubcommands.forceOverwrite) {
+                val existingOutputFiles = outputFiles.filter { it.exists() }
+                if (existingOutputFiles.isNotEmpty()) {
+                    throw UsageError("None of the output files $existingOutputFiles must exist yet.", statusCode = 2)
```

Comment:
This moves this check to after parsing the input file and running the evaluator, which can take some time for large result files. I prefer the command to fail early like it was before.

Reply:
Done.